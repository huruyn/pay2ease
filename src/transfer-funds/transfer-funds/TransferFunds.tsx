import {
  Button,
  Col,
  Form,
  Row,
} from 'react-bootstrap';
import CommonPageContainer from '../../common/CommonPageContainer';

const Divider = () => <div className="border-top my-3" />;

const LeftCol = () => (
  <Col xs={9}>
    <Row>
      <Col>
        <b>Step: 1 - 2</b>
      </Col>
    </Row>
    <Row>
      <Col>
        <h4 className="mt-4">Set up transfer details</h4>
        <p>Enter your transfer details</p>
        <p>
          Can&apos;t find your account in the list? Select &quot;Link Account&quot; below the
          account you would like to add to your transfer list on the My Accounts page.
        </p>
      </Col>
    </Row>
    <Row className="align-items-center mt-4">
      <Col xs={3}>From:</Col>
      <Col xs={6}>
        <Form.Select>
          <option>Choose an account</option>
        </Form.Select>
      </Col>
    </Row>
    <Divider />
    <Row className="align-items-center mt-4">
      <Col xs={3}>To:</Col>
      <Col xs={6}>
        <Form.Select>
          <option>Choose an account</option>
        </Form.Select>
      </Col>
    </Row>
    <Divider />
    <Row className="align-items-center">
      <Col xs={3}>Amount:</Col>
      <Col xs={6}>
        <Form.Control type="number" step=".01" placeholder="$" />
      </Col>
    </Row>
    <Divider />
    <Row className="align-items-center mt-4">
      <Col xs={3}>How often:</Col>
      <Col xs={6}>
        <Form.Select>
          <option>Once</option>
        </Form.Select>
      </Col>
    </Row>
    <Divider />
    <Row className="align-items-center">
      <Col xs={3}>
        Transfer date:
      </Col>
      <Col xs={6}>
        <Row>
          <Col xs="auto">
            <Form.Select>
              <option>November</option>
            </Form.Select>
          </Col>
          <Col xs="auto">
            <Form.Select>
              <option>9</option>
            </Form.Select>
          </Col>
          <Col xs="auto">
            <Form.Select>
              <option>2021</option>
            </Form.Select>
          </Col>
        </Row>
      </Col>
    </Row>
    <Divider />
    <Row>
      <Col className="text-end">
        <Button>Next</Button>
      </Col>
    </Row>
    <Divider />
    <Row>
      <Col>
        <b>Note:</b>
        <br />
        <ul>
          <li>
            For post-dated and recurring transfers, funds must be available in your account 1
            business day before the scheduled transfer date; otherwise the transaction may be
            cancelled without notice to you.
          </li>
          <li>
            Transfer requests submitted after 6 p.m. ET will be processed the next business day.
          </li>
          <li>
            The &quot;Account Balance&quot; that is displayed may include any funds on hold. The
            &quot;Available Funds&quot; shows how much money is available immediately to make a
            transfer.
          </li>
        </ul>
      </Col>
    </Row>
  </Col>
);

const RightCol = () => (
  <Col xs={3}>
    <div className="border p-2 rounded">
      <b>You can also:</b>
      <br />
      Contribute to a TFSA &gt;
      <br />
      Widthdraw from a TFSA &gt;
      <br />
      Contribute to an RRSP &gt;
      <br />
      Make an additional loan payment &gt;
      <br />
      Make a mortgage payment &gt;
    </div>
  </Col>
);

export default function TransferFunds() {
  return (
    <div className="page-content">
      <CommonPageContainer title="Transfers">
        <Row>
          <LeftCol />
          <RightCol />
        </Row>
      </CommonPageContainer>
    </div>
  );
}
