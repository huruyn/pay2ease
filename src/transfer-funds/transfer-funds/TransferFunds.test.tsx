import React from 'react';
import TransferFunds from './TransferFunds';

const ReactTestRenderer = require('react-test-renderer');

describe('TransferFunds Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(
      <TransferFunds />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
