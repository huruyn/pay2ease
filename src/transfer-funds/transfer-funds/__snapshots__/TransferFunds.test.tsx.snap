// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransferFunds Component matches the snapshot 1`] = `
<div
  className="page-content"
>
  <div
    className="d-flex justify-content-between align-items-center"
  >
    <h3>
      Transfers
    </h3>
  </div>
  <div
    className="content-wrapper"
  >
    <div
      className="row"
    >
      <div
        className="col-9"
      >
        <div
          className="row"
        >
          <div
            className="col"
          >
            <b>
              Step: 1 - 2
            </b>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col"
          >
            <h4
              className="mt-4"
            >
              Set up transfer details
            </h4>
            <p>
              Enter your transfer details
            </p>
            <p>
              Can't find your account in the list? Select "Link Account" below the account you would like to add to your transfer list on the My Accounts page.
            </p>
          </div>
        </div>
        <div
          className="align-items-center mt-4 row"
        >
          <div
            className="col-3"
          >
            From:
          </div>
          <div
            className="col-6"
          >
            <select
              className="form-select"
            >
              <option>
                Choose an account
              </option>
            </select>
          </div>
        </div>
        <div
          className="border-top my-3"
        />
        <div
          className="align-items-center mt-4 row"
        >
          <div
            className="col-3"
          >
            To:
          </div>
          <div
            className="col-6"
          >
            <select
              className="form-select"
            >
              <option>
                Choose an account
              </option>
            </select>
          </div>
        </div>
        <div
          className="border-top my-3"
        />
        <div
          className="align-items-center row"
        >
          <div
            className="col-3"
          >
            Amount:
          </div>
          <div
            className="col-6"
          >
            <input
              className="form-control"
              placeholder="$"
              step=".01"
              type="number"
            />
          </div>
        </div>
        <div
          className="border-top my-3"
        />
        <div
          className="align-items-center mt-4 row"
        >
          <div
            className="col-3"
          >
            How often:
          </div>
          <div
            className="col-6"
          >
            <select
              className="form-select"
            >
              <option>
                Once
              </option>
            </select>
          </div>
        </div>
        <div
          className="border-top my-3"
        />
        <div
          className="align-items-center row"
        >
          <div
            className="col-3"
          >
            Transfer date:
          </div>
          <div
            className="col-6"
          >
            <div
              className="row"
            >
              <div
                className="col-auto"
              >
                <select
                  className="form-select"
                >
                  <option>
                    November
                  </option>
                </select>
              </div>
              <div
                className="col-auto"
              >
                <select
                  className="form-select"
                >
                  <option>
                    9
                  </option>
                </select>
              </div>
              <div
                className="col-auto"
              >
                <select
                  className="form-select"
                >
                  <option>
                    2021
                  </option>
                </select>
              </div>
            </div>
          </div>
        </div>
        <div
          className="border-top my-3"
        />
        <div
          className="row"
        >
          <div
            className="text-end col"
          >
            <button
              className="btn btn-primary"
              disabled={false}
              type="button"
            >
              Next
            </button>
          </div>
        </div>
        <div
          className="border-top my-3"
        />
        <div
          className="row"
        >
          <div
            className="col"
          >
            <b>
              Note:
            </b>
            <br />
            <ul>
              <li>
                For post-dated and recurring transfers, funds must be available in your account 1 business day before the scheduled transfer date; otherwise the transaction may be cancelled without notice to you.
              </li>
              <li>
                Transfer requests submitted after 6 p.m. ET will be processed the next business day.
              </li>
              <li>
                The "Account Balance" that is displayed may include any funds on hold. The "Available Funds" shows how much money is available immediately to make a transfer.
              </li>
            </ul>
          </div>
        </div>
      </div>
      <div
        className="col-3"
      >
        <div
          className="border p-2 rounded"
        >
          <b>
            You can also:
          </b>
          <br />
          Contribute to a TFSA &gt;
          <br />
          Widthdraw from a TFSA &gt;
          <br />
          Contribute to an RRSP &gt;
          <br />
          Make an additional loan payment &gt;
          <br />
          Make a mortgage payment &gt;
        </div>
      </div>
    </div>
  </div>
  <div
    className="footer-content-wrapper"
  >
    <div
      className="quick-links container"
    >
      <div
        className="row"
      >
        <h3
          className="col-sm-12"
        >
          Quick Links
        </h3>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-2"
        >
          <ul>
            <li>
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li>
              Please Review Our Legal Stuff by visiting our
               
              <a
                className="text-decoration-none"
                href="/legal"
                onClick={[Function]}
              >
                Privacy Policy | TOU | EULA | CardHolder T&C Pages
              </a>
            </li>
          </ul>
        </div>
        <div
          className="col-sm-2"
        >
          8465 W Sahara Ave. Las Vegas, NEVADA
          <ul>
            <li>
              <a
                href="tel:+17026231876"
              >
                1.702.623.1876
              </a>
            </li>
            <li>
              <a
                href="mailto:contact@zippy.cash"
              >
                contact@zippy.cash
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12 col-sm-12 col-xs-12 text-left"
        >
          © 2021 Zippy Cash LLC  All rights reserved.
          <br />
          <br />
          The Program Manager for the Zippy Cash® Prepaid Visa Debit Card is PayQuicker, LLC
          <br />
          Zippy Cash® is a registered trademark of Zippy Cash, LLC
          <br />
          The Zippy Cash Prepaid Visa Debit Card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Visa U.S.A. Inc. “Metropolitan Commercial Bank” and “Metropolitan” are registered trademarks of Metropolitan Commercial Bank ©2014. BY USING THIS CARD YOU AGREE TO THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. All trademarks and service marks are the property of their respective owners.
          <br />
          In compliance with the USA PATRIOT Act* and other laws, anyone who is a US resident over 18 with a valid Social Security number and a verifiable physical US address (no PO boxes or APO addresses) can apply for an Zippy Cash Card. This information must be provided, along with other information pertinent to receiving approval, when applying for a Zippy Cash Card. Currently, we do not issue the Zippy Cash Card to residents of Puerto Rico, Guam, the Virgin Islands, American Samoa, or other US territories.
          <br />
          *Important information for opening a card account: To help the federal government fight the funding of terrorism and money laundering activities, the USA PATRIOT Act requires all financial institutions and their third parties to obtain, verify and record information that identifies each person who opens a card account. What this means for you: When you open a card account, we will ask for your name, address, date of birth, social security number or country identification number, and other information that will allows us to identify you. We may also request a copy of your driver’s license or other documentation bearing your photo as verification of your identity.
          <br />
          Apple and the Apple logo are trademarks of Apple Inc., registered in the U.S. and other countries. App Store is a service mark of Apple Inc. Google Play is a trademark of Google Inc. Data rates may apply. Samsung Pay is a trademark of Samsung Inc.
        </div>
      </div>
    </div>
    <footer
      className="container mb-3"
    >
      <div
        className="d-none d-md-block"
      >
        <div
          className="d-flex justify-content-between border-bottom"
        />
        <div
          className="d-flex justify-content-between border-top"
        >
          <div
            className=""
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
          <div
            className="copyright mt-1"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
      <div
        className="d-block d-md-none"
      >
        <div
          className="d-flex border-bottom"
        >
          <div
            className="d-flex flex-column me-2"
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
        </div>
        <div
          className="d-flex justify-content-end border-top"
        >
          <div
            className="copyright mt-2 text-end small"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
